### DISCUSSION TITLE
Migrate a local deployment of langfuse from one computer to another

### DISCUSSION THREAD

**Discussion by nithin1995** [langfuse/langfuse#2595] [Support] [answered]
What is the recommended way to move the local deployment of langfuse from one laptop to another while retaining all the data?
Can I do a `docker commit` and `docker run` ?

By doing this way, will the future upgrades of langfuse docker images work seamlessly?

**Comment by marcklingen** [comment, marked as answer]
The docker container is stateless. You’d need to move the database and its content to the new machine and then start the Langfuse container on the new machine (container version new machine >= old machine).

**Comment by nithin1995** [comment]
Thanks ! I was able to migrate. But I forgot the password to my account that I was using in my old laptop, not able to sign in. Is there any way I can recover that account?

**Reply by marcklingen** [reply]
you can create a new account and then reassign the projects manually in the project_memberships table to the new user.id from the users table
